lib/rrd_fring.mli

```diff
@@ -11,21 +11,29 @@
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  * GNU Lesser General Public License for more details.
  *)
-(** Ring structures *)
- 
-type t = {
+(** Ring structures for RRAs
+    The values in the structures are bound to a range *)
+
+module BoundedFloat = Rrd_utils.BoundedFloat
+
+type t = private {
```

Comment:
Unfortunately this doesn't protect against changing the mutable variable, or the bigarray even though it is marked as private (it just prevents anyone from creating a completely new ring).
Can we hide the type completely?